Unable to install with Go 1.12.12 and 1.13.3

Hi, 
I get an error when I try to install lego from source:

<details><summary>Go 1.12.12</summary>

```console
jens@jens-thinkpad:~$ go version
go version go1.12.12 linux/amd64
jens@jens-thinkpad:~$ GO111MODULE=on go get -u github.com/go-acme/lego/cmd/lego
go: finding github.com/go-acme/lego/cmd/lego latest
go: finding github.com/go-acme/lego/cmd latest
go: finding github.com/rainycape/memcache latest
go: finding github.com/labbsr0x/bindman-dns-webhook/src/client latest
go: finding github.com/labbsr0x/bindman-dns-webhook/src latest
go: finding github.com/Azure/go-autorest/autorest/azure latest
go: finding golang.org/x/oauth2/clientcredentials latest
go: finding golang.org/x/crypto/ocsp latest
go: finding golang.org/x/net/idna latest
go: finding golang.org/x/net/publicsuffix latest
go: finding golang.org/x/oauth2 latest
go: finding golang.org/x/crypto latest
go: finding golang.org/x/net latest
go: finding github.com/namedotcom/go/namecom latest
go: finding github.com/namedotcom/go latest
go: finding github.com/timewasted/linode/dns latest
go: finding github.com/timewasted/linode latest
go: finding github.com/aws/aws-sdk-go/aws/session latest
go: finding github.com/aws/aws-sdk-go/aws latest
go: finding github.com/aws/aws-sdk-go/service/route53 latest
go: finding github.com/aws/aws-sdk-go/service latest
go: finding github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests latest
go: finding github.com/aliyun/alibaba-cloud-sdk-go/sdk latest
go: finding github.com/aliyun/alibaba-cloud-sdk-go latest
go: finding github.com/aws/aws-sdk-go/service/lightsail latest
go: finding github.com/aliyun/alibaba-cloud-sdk-go/services/alidns latest
go: finding github.com/aliyun/alibaba-cloud-sdk-go/services latest
go: finding github.com/aws/aws-sdk-go/aws/client latest
go: finding github.com/sacloud/libsacloud/api latest
go: finding github.com/sacloud/libsacloud/sacloud latest
go: finding github.com/OpenDNS/vegadns2client latest
go: finding github.com/iij/doapi latest
go: finding github.com/dnsimple/dnsimple-go/dnsimple latest
go: finding github.com/gophercloud/gophercloud/openstack/dns/v2/zones latest
go: finding github.com/gophercloud/gophercloud/openstack/dns/v2/recordsets latest
go: finding github.com/gophercloud/gophercloud/openstack/dns/v2 latest
go: finding github.com/gophercloud/gophercloud/openstack/dns latest
go: finding github.com/gophercloud/gophercloud/openstack latest
go: finding github.com/aws/aws-sdk-go/aws/request latest
go: finding golang.org/x/oauth2/google latest
go: finding github.com/ovh/go-ovh/ovh latest
go: finding github.com/ovh/go-ovh latest
go: finding github.com/aliyun/alibaba-cloud-sdk-go/sdk/auth/credentials latest
go: finding github.com/aliyun/alibaba-cloud-sdk-go/sdk/auth latest
go: finding github.com/Azure/azure-sdk-for-go/services/dns/mgmt/2017-09-01/dns latest
go: finding github.com/Azure/azure-sdk-for-go/services/dns/mgmt/2017-09-01 latest
go: finding github.com/Azure/azure-sdk-for-go/services/dns/mgmt latest
go: finding github.com/Azure/azure-sdk-for-go/services/dns latest
go: finding github.com/Azure/azure-sdk-for-go/services latest
go: finding google.golang.org/api/googleapi latest
go: finding google.golang.org/api/dns latest
go: finding github.com/iij/doapi/protocol latest
go: finding github.com/akamai/AkamaiOPEN-edgegrid-golang/edgegrid latest
go: finding github.com/akamai/AkamaiOPEN-edgegrid-golang/configdns-v1 latest
go: finding google.golang.org/api/option latest
go: finding github.com/oracle/oci-go-sdk/common latest
go: finding github.com/oracle/oci-go-sdk/dns latest
go: finding golang.org/x/net/context latest
go: finding github.com/transip/gotransip/domain latest
go: finding gopkg.in/ns1/ns1-go.v2 latest
# github.com/go-acme/lego/providers/dns/designate
go/pkg/mod/github.com/go-acme/lego@v2.7.2+incompatible/providers/dns/designate/designate.go:206:3: cannot use record.TTL (type int) as type *int in field value
```

</details>

```
...
# github.com/go-acme/lego/providers/dns/designate
go/pkg/mod/github.com/go-acme/lego@v2.7.2+incompatible/providers/dns/designate/designate.go:206:3: cannot use record.TTL (type int) as type *int in field value
```

same happens with go version 13.3:

<details><summary>Go 1.13.3</summary>

```console
jens@jens-thinkpad:~$ go version
go version go1.13.3 linux/amd64
jens@jens-thinkpad:~$ GO111MODULE=on go get -u github.com/go-acme/lego/cmd/lego
go: finding gopkg.in/ns1/ns1-go.v2 latest
go: finding golang.org/x/crypto latest
go: finding github.com/rainycape/memcache latest
go: finding golang.org/x/oauth2 latest
go: finding golang.org/x/net latest
go: finding github.com/timewasted/linode latest
go: finding github.com/ovh/go-ovh latest
go: finding github.com/namedotcom/go latest
go: finding github.com/OpenDNS/vegadns2client latest
go: finding github.com/iij/doapi latest
go: finding github.com/aliyun/alibaba-cloud-sdk-go latest
go: finding google.golang.org/genproto latest
go: finding golang.org/x/sys latest
go: finding github.com/jmespath/go-jmespath latest
go: finding github.com/modern-go/concurrent latest
go: finding github.com/kolo/xmlrpc latest
go: finding golang.org/x/time latest
go: finding github.com/golang/groupcache latest
# github.com/go-acme/lego/providers/dns/designate
go/pkg/mod/github.com/go-acme/lego@v2.7.2+incompatible/providers/dns/designate/designate.go:206:3: cannot use record.TTL (type int) as type *int in field value
```

</details>

--- a/docs/content/installation/_index.md
+++ b/docs/content/installation/_index.md
@@ -45,8 +45,8 @@ Requirements:
 - `go` v1.12+
 - environment variable: `GO111MODULE=on`
 
-To install from sources, just run:
+To install the latest development version from sources, just run:
 
 ```bash
-go get -u github.com/go-acme/lego/cmd/lego
+go get -u github.com/go-acme/lego/v3/cmd/lego
 ```

